docs/sources/helm-charts/mimir-distributed/release-notes/v5.1.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Grafana Mimir Helm chart version 5.1 release notes"
+menuTitle: "V5.1 release notes"
+description: "Release notes for Grafana Mimir Helm chart version 5.1"
+weight: 300
+---
+
+# Grafana Mimir Helm chart version 5.1 release notes
+
+Grafana Labs is excited to announce version 5.1 of the Grafana Mimir Helm chart, which is compatible with Grafana Mimir v2.10 or Grafana Enterprise Metrics (GEM) v2.10. The `mimir-distributed` Helm chart is the best way to install Mimir on Kubernetes.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For a comprehensive list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
+
+## Important changes
+
+This release only contains minor enhacements to the Helm chart itself, but updates the Mimir and GEM binaries.
+
+## Features and enhancements
+
+Notable enhancements are as follows:
+
+- Pod management policy is now configurable for store-gateways and compactors.
+- Nginx now listens on IPv6 pod interface as well if available.
+- Distributor and querier CPU utilization better in line with CPU request by using `GOMAXPROCS`.
+- Deployments now set `maxUnavailable` to 0 to avoid becoming unavailable during rollout.
```

Comment:
maybe a word of caution for people who have also set `maxSurge: 0`, in which case rollouts will stall?

Reply:
the error I get when both are 0 is 
```
Invalid value: intstr.IntOrString{Type:0, IntVal:0, StrVal:""}: may not be 0 when `maxSurge` is 0
```

Reply:
updated